.travis.yml

```diff
@@ -6,7 +6,7 @@ node_js:
 sudo: true
 
 env:
-  - CXX=g++-4.8 NODE_API_DB_NAME_TEST=wolox-training API_PAGE_LIMIT=1
+  - CXX=g++-4.8 NODE_API_DB_NAME_TEST=wolox-training API_PAGE_LIMIT=1 API_ALBUMS_ENDPOINT=https://jsonplaceholder.typicode.com/albums
```

Comment:
in a project, dont set the env vars here, do it in the CI UI / in the terminal. We dont want to have secrets in our git repositories.